### PR TITLE
feat: reintroduce IsZero method on windows

### PR DESCRIPTION
**Pull request by jlapacik**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

**Review by fchikwekwe — APPROVED** [review]
lgtm

**Comment by nathanielc** [comment]
Do we have any tests for this?

File: interval/window.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ func NewWindow(every, period, offset values.Duration) (Window, error) {
 	return w, nil
 }
 
+// IsZero checks if the window's every duration is zero
+func (w Window) IsZero() bool {
+	return w.every.IsZero()
+}
+
 func (w Window) isValid() error {
 	if w.every.IsZero() {
```
Comment (Bolladeen) [inline interval/window.go:59]: We could utilize the new isZero method in this if-statement, right?

Comment (Bolladeen) [inline interval/window.go:59]: We could utilize the new isZero method in this if-statement, right?

**Review by Bolladeen — APPROVED** [review]
Other than the one small comment, it looks good.

**Pull request merged**